target/ppc/translate/fp-impl.c.inc

```diff
@@ -1473,6 +1473,21 @@ static void gen_stfqx(DisasContext *ctx)
     tcg_temp_free_i64(t1);
 }
 
+static bool ea_calc(DisasContext *ctx, int ra, int r, TCGv displ, TCGv *ea)
```

Comment:
Maybe we should declare this method in `translate.c` and change `do_ldst` in this commit.